### PR TITLE
Bug fixes and improvements (related to windowcoverings)

### PR DESCRIPTION
**Pull request by RobinBol**
- Add invertWindowCoveringsTiltDirection setting
- Fix dim/windowcoverings system capability
- Add windowcoverings_tilt_set system capability
- Refactor composeCustomSaveMessage to enable custom save messages for non-battery devices
- Add error logging

### CODE REVIEW

File: lib/zwave/ZwaveDevice.js [hunk 1]
```diff
@@ -753,26 +753,40 @@ class ZwaveDevice extends MeshDevice {
...
- 						case 2: payload.Properties1['Key Attributes'] = 'Key Held Down'; break;
- 						case 3: payload.Properties1['Key Attributes'] = 'Key Pressed 2 times'; break;
- 						case 4: payload.Properties1['Key Attributes'] = 'Key Pressed 3 times'; break;
- 						case 5: payload.Properties1['Key Attributes'] = 'Key Pressed 4 times'; break;
- 						case 6: payload.Properties1['Key Attributes'] = 'Key Pressed 5 times'; break;
- 					}
- 				}
- 			}
+			// Catch central scene echos and (sometimes) failing parser
+			if (command.name === 'CENTRAL_SCENE_NOTIFICATION') {
```
Comment (WeeJeWel) [inline lib/zwave/ZwaveDevice.js]: Why not put this in a specific Command Class file? `/lib/zwave/system/commandclasses/`

Comment (caseda) [reply lib/zwave/ZwaveDevice.js]: Cause that didn't exist when I added this few months ago :smile:

Comment (WeeJeWel) [reply lib/zwave/ZwaveDevice.js]: Fair enough. Can you split it up? I think command class specific code doesn't belong in the base class.

Comment (RobinBol) [reply lib/zwave/ZwaveDevice.js]: The command class code wasn't added with this commit (only its style was fixed). I suggest to merge this PR and refactor the central scene parsers as suggested by @WeeJeWel in another PR.

**Comment by WeeJeWel** [comment]
@RobinBol Can you resolve the conflict?

**Comment by RobinBol** [comment]
Done!

**Pull request merged**